src/qt/coincontroldialog.cpp

```diff
@@ -441,7 +441,7 @@ void CoinControlDialog::updateLabels(WalletModel *model, QDialog* dialog)
     for (const auto& out : model->wallet().getCoins(vCoinControl)) {
         if (out.depth_in_main_chain < 0) continue;
 
-        // unselect already spent, very unlikely scenario, this could happen
+        // deselect already spent, very unlikely scenario, this could happen
```

Comment:
If unselect is in the ignore list, why is this being changed?

Reply:
Good point. Fixed!